SPECS/kata-containers/kata-containers.spec

```diff
@@ -21,21 +21,16 @@
 %global kataclhdir              /usr/share/cloud-hypervisor
 %global katainitrddir           /var/cache/kata-containers/osbuilder-images/kernel-uvm
 
-%global runtime_make_vars       QEMUPATH=%{qemupath} \\\
-                                KERNELTYPE="compressed" \\\
+%global runtime_make_vars       KERNELTYPE="compressed" \\\
                                 KERNELPARAMS="systemd.legacy_systemd_cgroup_controller=yes systemd.unified_cgroup_hierarchy=0" \\\
-                                DEFSHAREDFS="virtio-fs" \\\
-                                DEFVIRTIOFSDAEMON=%{_libexecdir}/"virtiofsd" \\\
-                                DEFVIRTIOFSCACHESIZE=0 \\\
-                                DEFSANDBOXCGROUPONLY=false \\\
+                                DEFVIRTIOFSDAEMON=%{_libexecdir}/"virtiofsd-rs" \\\
+                                DEFSTATICRESOURCEMGMT_CLH=true \\\
```

Comment:
Thought: We should align DEFSTATICRESOURCEMGMT between Kata-CC and Kata: We do static allocation for both. For Kata-CC, we do not seem to set the variable. It would be cleaner to make this consistent.

Maybe this is yet to come since this is a draft.

Reply:
I don't set this in the CC spec because the equivalent CC flag upstream is `DEFSTATICRESOURCEMGMT_TEE` and it defaults to true 🙂